pkg/config/apiserver.go

```diff
@@ -144,9 +141,10 @@ func getRules(store KeyValueStore, path string) (statusCode int, msg string, dat
 
 	m := &pb.ServiceConfig{}
 	if err := yaml.Unmarshal([]byte(val), m); err != nil {
+		glog.Warningf("rules %s could not be parsed %s", path, err.Error())
 		return http.StatusInternalServerError, fmt.Sprintf("unbale to parse rules at %s", path), nil
```

Comment:
unbale -> unable

I think we also need to return the error to the caller. As it is, if the caller has a malformed parameter, the only info we tell them is "sorry, can't parse" with no indication of what the problem is.